src/main.ts

```diff
@@ -110,6 +111,8 @@ const generateList = async () => {
 
 const generateInfo = () => core.group('pixi info', () => execute(pixiCmd('info')))
```

Comment:
Are we printing anything inside of the group? If not, it would probably better be a core.info. If yes, this is fine.

Reply:
Yup, we're logging the path components being set along with the exported variables